Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -6,9 +6,10 @@ import Result
 import Stripe
 
 public protocol AddNewCardViewModelInputs {
+  func cardBrandIsValid(_ valid: Bool)
```

Comment:
I'm not really sure what kind of naming conventions have we been trying to follow or if there has been a big focus on this but we could do something less verbose (more Swift 3.x+).

i.e. `cardBrand(isValid: Bool)`

we do use a similar method bellow `paymentInfo(valid: Bool)`

What do you think?